### PR TITLE
Grid/Tile Icons settings Automation

### PR DESCRIPTION
**Pull request by akarol**

### CODE REVIEW

File: cfme/configure/settings.py [hunk 1]
```diff
@@ -111,12 +115,35 @@ class Visual(Updateable):
...
+            ('datastore_quad', "//input[@id='quadicons_storage']"),
+            ('datastoreitem_quad', "//input[@id='quadicons_storageitem']"),
+            ('vm_quad', "//input[@id='quadicons_vm']"),
+            ('vmitem_quad', "//input[@id='quadicons_vmitem']"),
+            ('template_quad', "//input[@id='quadicons_miq_template']"),
+        ])
+
     save_button = form_buttons.FormButton("Add this Time Profile")
 
-    def __init__(self, grid_view=None, tile_view=None, list_view=None):
```
Comment (psav) [inline cfme/configure/settings.py]: Same comment here as in #1482

File: cfme/web_ui/__init__.py [hunk 2]
```diff
@@ -2039,6 +2039,15 @@ def name(self):
         """ Returns name of the quadicon."""
         return self._name
 
+    @property
```
Comment (psav) [inline cfme/web_ui/__init__.py]: I'm not 100% sold on the name of this method. Perhaps if I am understanding it correctly, we could call it, ```icon_only``` or ```center_icon_only```.

**Comment by dajoRH** [comment]
Lint report for commit 33c02ff4606bf35498d57c9995db169c33a9a94e:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit b7460857ec54f6c070f246a8f62201c6b9c26571:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 93013621fd43eb6fa676ab1f4a2f2e6403811299:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 36e053fb38751c36f447b79be15c4378f203c788:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please?
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 5187158b41b53ac7e2eab556f5d98b86a5813a43:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/web_ui/__init__.py (hunk 2 above)
Comment (seandst) [inline cfme/web_ui/__init__.py]: I'm still not entirely sure what it does.

Can we get a docstring on this?

File: cfme/configure/settings.py [hunk 3]
```diff
@@ -166,4 +187,34 @@ def login_page(self, value):
...
+                                   'datastore_quad': updates.get('datastore_quad'),
+                                   'datastoreitem_quad': updates.get('datastoreitem_quad'),
+                                   'vm_quad': updates.get('vm_quad'),
+                                   'vmitem_quad': updates.get('vmitem_quad'),
+                                   'template_quad': updates.get('template_quad')})
+        sel.click(form_buttons.save)
+
+    def uncheck_checkboxes(self):
+        sel.force_navigate("my_settings_visual")
+        fill(self.quadicons_form, {'infra_provider_quad': self.infra_provider_quad,
```
Comment (seandst) [inline cfme/configure/settings.py]: If this really is supposed to uncheck all the checkboxes, shouldn't all of the values be `False`?

**Comment by mfalesni** [comment]
Lint report for commit 8a06d66e12f830f2c69a7ba70a6cce392632211c:
:hurtrealbad: There were some flake issues that need to be resolved in order to merge the pull request:
```
./cfme/tests/infrastructure/test_datastore_analysis.py:8:1: F401 'setup_provider' imported but unused
./cfme/tests/infrastructure/test_datastore_analysis.py:32:101: E501 line too long (109 > 100 characters)
./cfme/tests/infrastructure/test_datastore_analysis.py:62:1: F811 redefinition of unused 'setup_provider' from line 8
```
*CFME QE Bot*

File: cfme/web_ui/__init__.py (hunk 2 above)
Comment (dajohnso) [inline cfme/web_ui/__init__.py]: Aziza, in the doc string, can you change to something closer to...
Checks if the quad icon is a single quadrent icon which is present after changing to tile(?) view.

And lets change the name too to check_for_single_quadrent_icon, I believe that should clear it up

File: cfme/tests/configure/test_visuals.py [hunk 4]
```diff
@@ -92,3 +128,28 @@ def test_start_page(request, provider_init, start_page):
...
+    sel.force_navigate('infrastructure_providers')
+    assert visual.check_image_exists, "Image View Failed!"
+
+
+def test_host_noquads(request, provider_init, set_host_quad):
+    sel.force_navigate('infrastructure_hosts')
+    assert visual.check_image_exists, "Image View Failed!"
+
+
+def test_datastore_noquads(request, provider_init, set_datastore_quad):
```
Comment (dajohnso) [inline cfme/tests/configure/test_visuals.py]: This entire suite should lose provider_init and begin to use the global setup_provider fixture in fixtures/provider.py

Comment (dajohnso) [inline cfme/tests/configure/test_visuals.py]: Aziza, can we also get the lint errors cleaned up please

**Comment by dajoRH** [comment]
Lint report for commit 5a154ef6e2f2790819d15e56d3a482b3aa8948e8:
:hurtrealbad: There were some flake issues that need to be resolved in order to merge the pull request:
```
./cfme/tests/configure/test_visuals.py:10:1: F401 'testgen' imported but unused
```
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 0a3d1d2b31bf25ab86036951ae499f6ed4f6ee90:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/web_ui/__init__.py (hunk 2 above)
Comment (seandst) [inline cfme/web_ui/__init__.py]: s/quadrent/quadrant/

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please?
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit a9f3f694c7c3dda0c33c448f04671e06d98198a6:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**